modules/instance_template/README.md

```diff
@@ -17,6 +17,7 @@ See the [simple](../../examples/instance_template/simple) for a usage example.
 | additional\_disks | List of maps of additional disks. See https://www.terraform.io/docs/providers/google/r/compute_instance_template.html#disk_name | <pre>list(object({<br>    disk_name    = string<br>    device_name  = string<br>    auto_delete  = bool<br>    boot         = bool<br>    disk_size_gb = number<br>    disk_type    = string<br>  }))</pre> | `[]` | no |
 | auto\_delete | Whether or not the boot disk should be auto-deleted | `string` | `"true"` | no |
 | can\_ip\_forward | Enable IP forwarding, for NAT instances for example | `string` | `"false"` | no |
+| default\_local\_ssd\_interface | The default interface to be used for local-ssd disks | `string` | `"SCSI"` | no |
```

Comment:
I'd like to avoid making this into a new variable. Can we just choose a preferred default and stick to it?

Reply:
Sure, I added for flexibility but I think NVME should be the default as it is the most performant. Will push that change.